### PR TITLE
Refact/keyp 360 initstateservice, plugins update

### PR DESCRIPTION
**Pull request by jeanpierrefortune**

### CODE REVIEW

**Review by andrei-cristea — APPROVED** [review]

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/plugin/reader/AbstractLocalReader.java [hunk 1]
```diff
@@ -824,9 +824,13 @@ private ApduResponse case4HackGetResponse(int originalStatusCode) {
...
    * removal sequence, etc.)
    *
+   * <p>The reader will remain in the WAIT_FOR_SE_REMOVAL state as long as the card is present. It
+   * will change to the WAIT_FOR_START_DETECTION or WAIT_FOR_SE_INSERTION state depending on what
+   * was set when the detection was started.
+   *
    * @since 0.9
    */
-  abstract void terminateCardCommunication();
+  abstract void finalizeCardProcessing();
```
Comment (pterr) [inline java/component/keyple-core/src/main/java/org/eclipse/keyple/core/plugin/reader/AbstractLocalReader.java:833]: No, this new name is too close to ObservableReader.finalizeCardProcessing()!

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/service/util/ContactsCardCommonProtocols.java [hunk 2]
```diff
@@ -17,11 +17,11 @@
  * @since 1.0
  */
 public enum ContactsCardCommonProtocols {
```
Comment (pterr) [inline java/component/keyple-core/src/main/java/org/eclipse/keyple/core/service/util/ContactsCardCommonProtocols.java:19]: Rename ContactsCardCommonProtocols to ContactCardCommonProtocols

**Comment by pterr** [comment]
Please no merge to develop, pending the finalization of the release 1.0.0 alpha 1

**Review by andrei-cristea — APPROVED** [review]

**Review by andrei-cristea — APPROVED** [review]

**Review by ocfmem — APPROVED** [review]

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/plugin/reader/ObservableReaderStateService.java [hunk 3]
```diff
@@ -12,41 +12,120 @@
...
+    if (reader instanceof WaitForCardRemovalDuringProcessing) {
+      final SmartRemovalMonitoringJob smartRemovalMonitoringJob =
+          new SmartRemovalMonitoringJob((WaitForCardRemovalBlocking) reader);
+      this.states.put(
+          AbstractObservableState.MonitoringState.WAIT_FOR_SE_PROCESSING,
+          new WaitForCardProcessingState(
+              this.reader, smartRemovalMonitoringJob, this.executorService));
+    } else {
+      this.states.put(
+          AbstractObservableState.MonitoringState.WAIT_FOR_SE_PROCESSING,
```
[the remark this answers is not in the document]
Comment (jeanpierrefortune) [reply java/component/keyple-core/src/main/java/org/eclipse/keyple/core/plugin/reader/ObservableReaderStateService.java:99]: I agree.
The most complicated thing is to find an elegant negation of WaitForCardRemovalDuringProcessing!
DontWaitForCardRemovalDuringProcessing?

File: java/component/keyple-core/src/main/java/org/eclipse/keyple/core/service/util/ContactsCardCommonProtocols.java (hunk 2 above)
Comment (jeanpierrefortune) [reply java/component/keyple-core/src/main/java/org/eclipse/keyple/core/service/util/ContactsCardCommonProtocols.java:19]: Yes.

**Review by andrei-cristea — APPROVED** [review]

**Review by pterr — APPROVED** [review]

**Pull request merged**